tools/publish/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+load("//python:pip.bzl", "compile_pip_requirements")
+
+compile_pip_requirements(
+    name = "requirements",
+    requirements_windows = "requirements_windows.txt",
+    # This fails on RBE and MacOS right now, and we don't need coverage there
+    tags = ["manual"],
```

Comment:
Could you remove this? I landed a fix here: https://github.com/bazelbuild/rules_python/commit/e7b51d7e557a2a5a244c58a81f77b5ab090f4973.

Reply:
it will still fail, I'll comment out this line so you see the failure